### PR TITLE
add kornia in the used by readme

### PR DESCRIPTION
**Pull request by edgarriba**

### CODE REVIEW

**Comment by Caglar1881** [comment]
#244

File: README.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ A ferret is actually a really good spirit animal for this project: cute, small,
 - [psycopg3]
 - [pip]
 - [packaging]
+- [kornia](https://kornia.readthedocs.io/en/latest/) -- Computer Vision library for PyTorch
```
Comment (Caglar1881) [inline README.md:89]: @@ -1,91 +1,92 @@

**Review by Caglar1881 — COMMENTED** [review]
@@ -1,91 +1,92 @@

**Comment by pradyunsg** [comment]
Hi! I'm gonna have to say no to this PR -- I've been curating that list myself, and I'd like to keep that as short as possible. I do appreciate the PR though!

Thanks for the suggestion! :)

**Pull request closed without merging**